src/test/java/com/solvd/CartTest.java

```diff
@@ -0,0 +1,65 @@
+package com.solvd;
+
+import com.solvd.pages.common.CartPageBase;
+import com.solvd.pages.common.HomePageBase;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import static com.solvd.ProductService.*;
+
+
+public class CartTest extends AbstractTest {
+    //    CID1
+    @Test
+    public void testDisplayChosenProductInCart() {
+        HomePageBase homePage = initPage(driver, HomePageBase.class);
+        homePage.open();
+
+        homePage.addProductToCart(SKINSHEEN_BRONZER_STICK.getIndex());
+        CartPageBase cartPage = homePage.clickCart();
+        Assert.assertEquals(cartPage.getProductName(0), SKINSHEEN_BRONZER_STICK.getName(), "Failed to display correct product");
+        Assert.assertEquals(cartPage.getProductQuantity(0), "1", "Failed to display correct product quantity");
+    }
+
+//    CID2
+    @Test
+    public void testDisplayDifferentChosenProductsInCart() {
+        HomePageBase homePage = initPage(driver, HomePageBase.class);
+        homePage.open();
+
+        homePage.addProductToCart(SKINSHEEN_BRONZER_STICK.getIndex());
+        homePage.addProductToCart(TREATMENT_SPF_15.getIndex());
```

Comment:
Calling addProductToCart multiple times is redundant, in case you need to add 10 product you will create addProductToCart 10 times?